Libplanet.Standalone/Hosting/LibplanetNodeService.cs

```diff
@@ -120,14 +120,17 @@ public class LibplanetNodeService<T> : IHostedService, IDisposable
             );
             _minerLoopAction = minerLoopAction;
             _exceptionHandlerAction = exceptionHandlerAction;
+            var rand = new Random();
+            IEnumerable<IceServer> shuffledIceServers = 
```

Comment:
I guess it can crash when `iceServers` is `null`. (see also. https://github.com/planetarium/NineChronicles.Standalone/pull/124/checks?check_run_id=1319321725)